projects/plugins/jetpack/tools/check-block-assets.php

```diff
@@ -86,6 +86,9 @@
 		'wp-plugins',
 		'wp-primitives',
 	),
+	'donations'      => array(
```

Comment:
CI wants 'donations' to be added in between 'ai-chat' and 'podcast-player' rather than at the end.

Reply:
yep. Looks like. updated